app/views/schools/register_ect_wizard/previous_ect_details.html.erb

```diff
@@ -0,0 +1,47 @@
+
+<% page_data(
+    title: "#{@ect.full_name} has been registered before",
+    error: @wizard.current_step.errors.present?,
+    backlink_href: @wizard.previous_step_path
+) %>
+
+<%= govuk_summary_list(card: { title: 'Previously registered details' } ) do |list| %>
+  <% list.with_row do |row| %>
+    <%= row.with_key { "School name" } %>
+    <%= row.with_value { Teachers::CurrentSchools.new(trn: @ect.trn).ect_at&.name || 'Not confirmed' } %>
```

Comment:
`<%= row.with_value { @ect.previous_school_name || 'Not confirmed' } %>`

Reply:
Thank you! Updated here: https://github.com/DFE-Digital/register-early-career-teachers-public/pull/554/commits/ae852762bf3590abca5202342b9d115537355866